nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -219,6 +219,13 @@
   private final Map<Symbol.ClassSymbol, Multimap<Tree, Element>> initTree2PrevFieldInit =
       new LinkedHashMap<>();
 
+  /**
+   * dynamically computer/overriden nullness facts for certain expressions, such as specific method
+   * calls where we can infer a more precise set of facts than those given by the method's
+   * annotations.
+   */
+  private final Map<ExpressionTree, Nullness> computedNullnessMap = new LinkedHashMap<>();
```

Comment:
Let's clear this map inside `matchClass` where we clear the other state

Reply:
Done.